src/test/java/com/iota/iri/network/UDPNeighborTest.java

```diff
@@ -0,0 +1,52 @@
+package com.iota.iri.network;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetSocketAddress;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UDPNeighborTest {
+
+    private static final String HOST = "localhost";
+    private static final int PORT = 42;
+
+    @Mock
+    private DatagramSocket udpSocket;
+
+    private final DatagramPacket packet = new DatagramPacket(new byte[]{}, 0, 0);
+
+    @Test
+    public void whenSendThenSendPacketToSocket() throws IOException {
+        UDPNeighbor neighbor = new UDPNeighbor(new InetSocketAddress(HOST, PORT), udpSocket, true);
+        neighbor.send(packet);
+        verify(udpSocket).send(packet);
+    }
+
+    @Test
+    public void whenSendExceptionThenCatchAndContinue() throws IOException {
```

Comment:
So when does the test fail?

Consider a solution like this:
https://stackoverflow.com/questions/2469911/how-do-i-assert-my-exception-message-with-junit-test-annotation

Reply:
It will never fail as there is a try catch in the code. I just used this method to check the log message. I am going to remove the tests or feel free to do so.